apps/cow-fi/components/SearchBar/styled.ts

```
@@ -0,0 +1,261 @@
+import styled from 'styled-components/macro'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)